### PR TITLE
smallcaps followup changes

### PR DESCRIPTION
**Pull request by warner**
address some additional review feedback from PR #7585

### CODE REVIEW

**Comment by FUDCo** [comment]
Didn't #7595 already do the `toCapData` et al thing?

File: packages/governance/test/unitTests/test-committee.js [hunk 1]
```diff
@@ -82,7 +82,7 @@ test('committee-open question:one', async t => {
     maxChoices: 1,
     maxWinners: 1,
     closingRule: {
-      timer: harden(buildManualTimer(t.log)),
+      timer: buildManualTimer(t.log),
```
Comment (erights) [inline packages/governance/test/unitTests/test-committee.js]: Still should have `harden` on line 77 above.

**Review by erights — APPROVED** [review]
LGTM, thanks!

Consider staging on (or waiting for) @turadg 's PR for some of these changes, to reduce the redundancy.

**Review by turadg — APPROVED** [review]
the other followup PR had an overzealous find/replace. happy for this one to get in first and I'll get to the other eventually

**Comment by mhofman** [comment]
@mergifyio refresh

File: packages/internal/src/storage-test-utils.js [hunk 2]
```diff
@@ -133,7 +139,10 @@ export const makeMockChainStorageRoot = () => {
   return Far('mockChainStorage', {
     ...bindAllMethods(rootNode),
     /**
-     * Defaults to deserializing pass-by-presence objects into Remotable objects.
+     * Defaults to deserializing slot references into plain Remotable
+     * objects, but if supplied with a different marshaller, it could
+     * produce Remotables with e.g. the slog string embedded in the
+     * iface.
```
Comment (gibson042) [inline packages/internal/src/storage-test-utils.js:145]:
```suggestion
     * Defaults to deserializing slot references into plain Remotable
     * objects having the specified interface name (as from `Far(iface)`),
     * but can accept a different marshaller for producing Remotables
     * that e.g. embed the slot string in their iface name.
```

Comment (warner) [reply packages/internal/src/storage-test-utils.js:145]: works for me, but the PR landed already.. @turadg did you have more followup to do? wanna include this in yours?

Comment (warner) [reply packages/internal/src/storage-test-utils.js:145]: oh nevermnd, I see #7610

**Pull request merged**